flight/UAVObjects/uavobjectmanager.c

```diff
@@ -377,7 +397,7 @@ UAVObjHandle UAVObjRegister(uint32_t id,
  */
 UAVObjHandle UAVObjGetByID(uint32_t id)
 {
-	UAVObjHandle * found_obj = (UAVObjHandle *) NULL;
+	UAVObjHandle found_obj = NULL;
```

Comment:
Took me a second to figure out what was going on here. It's a little confusing (for me) to have a typedef be a pointer (```typedef struct UAVOBase * UAVObjHandle```).